util/pom.xml

```diff
@@ -16,7 +16,6 @@
     <dependency>
       <groupId>org.agrona</groupId>
       <artifactId>agrona</artifactId>
-      <scope>provided</scope>
```

Comment:
❓ Was this on purpose? It's hidden in a `style` commit.

Reply:
I consider it a style, primarily because it was unnecessary - zeebe-util is not meant to be used outside of Zeebe, AFAIK, you could question why is this one provided but not the others? It's not an optional dependency at all, as it's used widely everywhere. In that sense, it has no impact as far as I can see for us, which is why I removed it.